### DISCUSSION TITLE
High CPU usage by Seq when idle

### DISCUSSION THREAD

**Discussion by kldjoannasandler** [datalust/seq-tickets#1902] [Help]
Hi there,

On one of our deployments of Seq is suddenly sitting between 60% and 90% CPU usage while effectively idle (events received and ingestion rates are less than 10 events/minute). Other instances of Seq we have deployed the same way on similar machines are showing less than 10% CPU usage when ingesting at similar rates, and and only getting into the 30s when ingesting thousands of events per minute.

This appears to be causing a huge slowdown for the UI and intermittent 502s for users.

The machine appears to be performing with normal disc read/write speeds from the diagnostic tests we ran, and doesn't have any other configuration changes that point to the sudden increase in CPU consumption. 

None of the Activity visible on the diagnostics page is anything unexpected, and we've retried restarting both Seq and the machine but neither of these solved the issue of high cpu usage.

Thanks in advance for any help that can be provided. We haven't been able to find any solutions so far from searching threads for similar discussions.

**Comment by KodrAus** [comment]
Hi @jsandlerkld 👋 

Would it be possible to send a diagnostic report through to our support@datalust.co address by visiting Settings > Diagnostics > Download a full diagnostic report?

**Reply by kldjoannasandler** [reply]
Yep, can do!
